packages/app/src/features/liquidity-flow/components/borrow-increase-modal/borrow-increase-modal.tsx

```diff
@@ -61,7 +61,7 @@ export function BorrowIncreaseModal({
   const { availableToBorrow } = useMarketDataForDisplay(tokenContracts[coreDebtTokenName])
 
   const inputDebtSumUsd =
-    Number(value) * coreDebtInfo.priceInUsd + Number(extraValue) * extraDebtInfo.priceInUsd
+    Number(value) * coreDebtInfo.priceInUsd + Number(extraValue) / extraDebtInfo.priceInUsd
```

Comment:
Don't we have to divide the left value as well?

Reply:
fixed

Reply:
Thank you.